README.md

```diff
@@ -1,6 +1,6 @@
 # loklak
 [![Build Status](https://travis-ci.org/loklak/loklak_server.svg?branch=master)](https://travis-ci.org/loklak/loklak_server)
-
+[![Join the chat at https://gitter.im/loklak/loklak](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/loklak/loklak)
 [![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy)
```

Comment:
Can you add a new line between these 2 lines, we can keep the deployment buttons in one line and statuses and tinier bars in one line, right now they all move into one line.